### ISSUE TITLE
Pagination failure of Service Events

### ISSUE THREAD

**Issue by johnrkriter** [kubernetes/dashboard#3202]
##### Environment

```
Dashboard version:v1.8.3 commit ec1d7de4456e6a397c7f931f0a2bfc74a6ca2e9c
Kubernetes version:V1.10(EKS)
Operating system:OSX 10.13.6
Node.js version:Unknown
Go version:Unknown
```

##### Steps to reproduce
Follow the 2 following AWS Tutorials for deploying an EKS cluster, then add nodes, and then deploy the master branch version of the Dashboard
https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html (Skip step 4, no apps on the cluster)
https://docs.aws.amazon.com/eks/latest/userguide/dashboard-tutorial.html

I then deployed Helm according to the following 
https://medium.com/@zhaimo/using-helm-to-install-application-onto-aws-eks-36840ff84555

I then deployed a helm chart for reactive-interaction-gateway, which caused a service to be created.
Helm chart here
https://github.com/Accenture/reactive-interaction-gateway/tree/master/deployment/helm

After changing the variables to use LoadBalancer in the helm chart and deploying it, I went to inspect the UI to show me status.

##### Observed result

After selecting the Service in UI, the Events list shows I have 15 events, displaying 1-10. I attempt to page so I can see 11-15 but I receive a 404

##### Expected result
UI should page the event list to show 11-15

**Comment by j-rewerts** [comment]
I'll see if I can reproduce this within Minikube.

**Comment by j-rewerts** [comment]
@johnrkriter Do you recall the config you passed to Helm?

The basic setup seems to be `helm install deployment/helm/reactive-interaction-gateway`

**Comment by johnrkriter** [comment]
I am travelling right now without my work laptop so I will need to check, however I was mostly following this guide https://gist.github.com/superseb/3a9c0d2e4a60afa3689badb1297e2a44
@j-rewerts

**Comment by floreks** [comment]
I can confirm that such issue exists. We are missing endpoint to handle this request on the backend. `api/v1/service/:namespace/:name/event`

**Comment by shu-mutou** [comment]
I could not represent this issue with minikube 0.33.1, kubernetes 1.13.2 and dashboard on master branch.

![screenshot from 2019-02-05 17-59-41](https://user-images.githubusercontent.com/12838129/52262753-b61de800-2970-11e9-8b16-3ee17dd73253.png)

Logs in dashboard:
```
[0] 2019/02/05 08:37:19 [2019-02-05T08:37:19Z] Incoming HTTP/1.1 GET /api/v1/service/default/hands-on-example-app/event?itemsPerPage=10&page=1 request from 127.0.0.1:55790: {}
[0] 2019/02/05 08:37:19 Found 10 events related to hands-on-example-app service in default namespace
[0] 2019/02/05 08:37:19 [2019-02-05T08:37:19Z] Outcoming response to 127.0.0.1:55790 with 200 status code
[0] 2019/02/05 08:37:21 [2019-02-05T08:37:21Z] Incoming HTTP/1.1 GET /api/v1/service/default/hands-on-example-app/event?itemsPerPage=10&page=2 request from 127.0.0.1:55796: {}
[0] 2019/02/05 08:37:21 Found 3 events related to hands-on-example-app service in default namespace
[0] 2019/02/05 08:37:21 [2019-02-05T08:37:21Z] Outcoming response to 127.0.0.1:55796 with 200 status code
```

@floreks I guess the endpoint for the service events is implemented as follow:
https://github.com/kubernetes/dashboard/blob/master/src/app/backend/resource/service/events.go

Please let me know if I misunderstood this issue. Thanks.

**Comment by floreks** [comment]
@shu-mutou You understood it correctly. It was fixed by #3153, but as it was part of angular migration, it is not available in the v1 version.

**Comment by maciaszczykm** [comment]
Great, thanks @shu-mutou.
